src/osg_display/gratia_datasource.py

```diff
@@ -17,40 +74,27 @@ def run(self):
         self.connect()
 
     def disconnect(self):
-        self.conn.close()
+        pass
 
-    def connect(self):
-        user = self.cp.get("Gratia", "User")
-        password = self.cp.get("Gratia", "Password")
-        host = self.cp.get("Gratia", "Host")
-        database = self.cp.get("Gratia", "Database")
-        port = int(self.cp.get("Gratia", "Port"))
+    def connect_gracc_url(self, gracc_url):
         try:
-            self.conn = MySQLdb.connect(user=user, passwd=password, host=host,
-                port=port, db=database)
-            log.info("Successfully connected to Gratia database")
+            self.es = elasticsearch.Elasticsearch(
+                [gracc_url], timeout=300, use_ssl=True, verify_certs=True,
+                ca_certs='/etc/ssl/certs/ca-bundle.crt')
         except Exception, e:
             log.exception(e)
-            log.error("Unable to connect to Gratia database")
+            log.error("Unable to connect to Gracc database")
```

Comment:
A careless mistake on my part here!  (I got the all caps right for the rest of the log messages! :)